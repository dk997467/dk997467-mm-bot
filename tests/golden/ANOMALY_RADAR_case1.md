ANOMALY RADAR

| bucket | kind | value | med | mad |
|--------|------|-------|-----|-----|
| 00:30 | EDGE | -1.000000 | 3.000000 | 0.100000 |
| 01:00 | TAKER | 30.000000 | 12.200000 | 0.100000 |

What to do:
- check edge breakdown: gross/fees/adverse/slippage on buckets
- re-check taker caps/routing and fees settings
