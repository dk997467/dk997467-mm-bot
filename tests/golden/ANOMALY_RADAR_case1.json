{"anomalies":[{"bucket":"00:30","kind":"EDGE","mad":0.100000,"med":3.000000,"value":-1.000000},{"bucket":"01:00","kind":"TAKER","mad":0.100000,"med":12.100000,"value":30.000000}],"bucket_minutes":15,"runtime":{"utc":"1970-01-01T00:00:00Z","version":"0.1.0"}}


